test/Icicle/Test/Core/Arbitrary.hs

```diff
@@ -437,6 +446,8 @@ baseValueForType t
      -> VBool <$> arbitrary
     TimeT
      -> VTime <$> arbitrary
+    FactIdentifierT
+     -> VFactIdentifier . FactIdentifier <$> arbitrary
```

Comment:
damn, good point. I guess this should fail, and perhaps FactIdentifierT shouldn't be generated in the type arbitrary either